lib/common/components/SelectFileModal.js

```diff
@@ -46,10 +51,11 @@ export default class SelectFileModal extends Component<Props, State> {
         onClose: props.onClose,
         onConfirm: props.onConfirm,
         showModal: true,
+        showMTCMergeWarning: false,
         title: props.title
       })
     } else {
-      this.setState({ disabled: false, showModal: true })
+      this.setState({ disabled: false, showModal: true, showMTCMergeWarning: false })
```

Comment:
Confused as to why we need this -- can't we just allow a default false?

Reply:
We needed this because we were setting the state on the opening of the modal, and we needed to set it for both cases of opening (props or no props). However, I think it makes more sense to just set the state on close, rather than open and then that takes care of that confusing line.